content/topics/kube2pulumi.md

```diff
@@ -0,0 +1,639 @@
+            converter doesn't yet support. To work with an engineer to help with your evaluation, please
+            <a href="{{< relref "/about#contactus" >}})" class="link">contact us</a> or
+            <a href="https://slack.pulumi.com" class="link">join our Community Slack</a>. We are here to help!
+        </p>
+    </div>
+</div>
+`;
+
+// Display the "could not convert" boilerplate.
+function displayCouldNotConvert(language) {
+    clearLanguageFiles(language);
+    $("#pulumi-code-"+language+"-files").html(couldNotConvertError);
+}
+
+// Now set up our event handler for conversion.
+function convertCode(language) {
+    // If we got called without an explicit language, look it up.
+    language = language || getCurrentLanguage();
+
+    // Get the currently chosen language by looking up the active language chooser tab.
+    let languageTextbox = language;
+    if (language === "c#") {
+        language = "csharp";
+        languageTextbox = "csharp";
+    } else if (language === "javascript") {
+        language = "typescript";
+        languageTextbox = "javascript";
+    }
+
+    // Clear the current fields.
+    $("#pulumi-errors").hide();
+    $("#pulumi-warnings").hide();
+    clearLanguageFiles(languageTextbox);
+
+    // Now read the various possible code sources.
+    let tfCode = $("#terraform-code").val();
+    let tfUrl = $("#terraform-url").val();
+    let tfUploadFiles = $("#terraform-upload")[0].files;
+    if (tfUrl === "https://") {
+        tfUrl = "";
+    }
+
+    // Read the input kind and verify that we've got what we need.
+    let tfIk = getCurrentInputKind();
+    switch (tfIk) {
+    case "url":
+        if (tfUrl === "") {
+            $("#pulumi-errors").text("Error: Please enter a URL for the code to convert above, and then try again");
+            $("#pulumi-errors").show();
+            return;
+        }
+        break;
+    case "upload":
+        if (!tfUploadFiles || !tfUploadFiles.length) {
+            $("#pulumi-errors").text("Error: Please choose code files to upload above, and then try again");
+            $("#pulumi-errors").show();
+            return;
+        }
+        break;
+    default: // "code"
+        if (tfCode === "") {
+            $("#pulumi-errors").text("Error: Please enter the code to convert above, and then try again");
+            $("#pulumi-errors").show();
+            return;
+        }
+        break;
+    }
+
+    // Add some "waiting" touches.
+    $(document.body).css({ "cursor": "wait" });
+    $("#terraform-code, #terraform-url").css({ "cursor": "wait" });
+    $("pulumi-chooser[type='language'] > ul > li > a").css({ "cursor": "wait" });
+    addLanguageFile(languageTextbox, "…", "…");
+
+    // Post to the endpoint and then, afterwards, add the result to the textbox.
+    let post = {
+        url: "https://k3wz4qdpci.execute-api.us-west-2.amazonaws.com/stage/convertKube",
```

Comment:
I will update this URL before merging once we get https://github.com/pulumi/home/pull/1042 deployed.